test/example_packages/with_makefile/Makefile

```diff
@@ -0,0 +1,9 @@
+INCLUDE_FLAGS = $(addprefix -I,$(INCLUDE_DIRS))
+
+$(BUILD_DIR)/libwith_makefile.a: $(BUILD_DIR)/hello_makefile.o
```

Comment:
I am guessing you cannot use a target like this on Windows.

Reply:
I suspect it's the `D:` at the front. Make sees that as the end of the target name. can you put quotes around that in a makefile?